GrandCommonDivisor.Core/Utils/GCDUtils.cs

```diff
@@ -0,0 +1,36 @@
+﻿using GrandCommonDivisor.Core.models;
+using System;
+
+namespace GrandCommonDivisor.Core
+{
+    public static class GCDUtils
+    {
+        public static int CalculateGCD(int firstN, int secondN)
+        {
+            if (secondN == 0)
+            {
+                return firstN;
+            }
+            int modulo = firstN % secondN;
```

Comment:
It is better usually to have an empty line after closing code section

Reply:
Fixed.